packages/utils/src/misc.ts

```diff
@@ -350,7 +350,14 @@ function _htmlElementAsString(el: unknown): string {
  * Returns a timestamp in seconds with milliseconds precision.
  */
 export function timestampWithMs(): number {
-  return new Date().getTime() / 1000;
+  // TODO: move this polyfill somewhere else
```

Comment:
Do we have a good place to put this polyfill?

Reply:
This is fine here!

Reply:
Here is not ideal because we get to test for it on every call to `timestampWithMs`, and that's every span.
Best to do it only once per page load / process.
Do we have a good place for those initialization run-once tasks?

Reply:
We already depend on `Date.now` in other places and I haven't realized that. Dropping the polyfill altogether.